webapp/packages/plugin-navigation-tree/src/NavigationTree/ElementsTree/ElementsTreeTools/ElementsTreeTools.tsx

```diff
@@ -67,13 +67,13 @@ export const ElementsTreeTools = observer<React.PropsWithChildren<Props>>(functi
 
   return styled(deprecatedStyles)(
     <SContext registry={registry}>
-      <tools className={s(styles, { tools: true })}>
-        <actions className={s(styles, { actions: true })}>
+      <div className={s(styles, { tools: true })}>
+        <div className={s(styles, { actions: true })}>
           {tree.settings?.configurable && (
             <IconButton
               name="/icons/settings_cog_sm.svg"
               title={translate('ui_settings')}
-              className={s(styles, { primary: true, opened })}
+              className={s(styles, { primary: true, actionIconButton: true, opened })}
```

Comment:
It's better to use SContext in such cases (you want to change the style of `IconButton` components inside actions div.